src/views/SetupKeys.tsx

```diff
@@ -96,23 +96,25 @@ export const SetupKeys = () => {
             message.loading({ content: 'Deleting...', key: deleteKey, style: styleNotification });
         } else if (deletedSetupKey.success) {
             message.success({ content: 'Setup key has been successfully removed.', key: deleteKey, duration: 2, style: styleNotification });
-            dispatch(setupKeyActions.setDeleteSetupKey({ ...deletedSetupKey, success: false }));
+            dispatch(setupKeyActions.setDeleteSetupKey({ ...deletedSetupKey, success: false }))
+            dispatch(setupKeyActions.resetDeletedSetupKey(null))
```

Comment:
Codacy has a fix for the issue: [Replace `······dispatch(setupKeyActions.resetDeletedSetupKey(null))` with `);⏎······dispatch(setupKeyActions.resetDeletedSetupKey(null));`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
      );
```